test/functional/tests/volumes/test_many_lvms_on_single_core.py

```diff
@@ -0,0 +1,113 @@
+#
+# Copyright(c) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import datetime
+
+import pytest
+
+from api.cas import casadm
+from api.cas.init_config import InitConfig
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from storage_devices.lvm import Lvm, LvmConfiguration, get_block_devices_list
+from test_tools import fs_utils
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import IoEngine, ReadWrite, VerifyMethod
+from test_utils.size import Size, Unit
+
+opencas_conf_path = "/etc/opencas/opencas.conf"
+lv_count = 16
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_many_lvms_on_single_core():
+    """
+        title: Test for LVM creation on CAS device - many lvms on single core.
+        description: |
+          Validation of LVM support, many LVMs created on CAS devices with single core:
+          1 cache, 1 core, 16 lvms.
```

Comment:
I'm not sure if this description is compatible with project vocabulary. Probably you mean 1 CAS device (core), 1 core device, 1 cache device and many LVMs on CAS device.